### ISSUE TITLE
Crash when assigning null to int in classes

### ISSUE THREAD

**Issue by NecroRomnt** [colobot/colobot#706]
Console log:

```

> [INFO]: Game paused - CBot editor
> colobot: /home/username/Загрузки (Downloads)/colobot-gold/src/colobot-colobot-gold-0.1.6-alpha/src/CBot/CBotVar.cpp:658: virtual void CBotVar::SetPointer(CBotVar*): Проверочное утверждение «0» не выполнено.
> Unhandled exception occured!
> ==============================
> SIGABRT, abort
> ==============================
> 
> This is usually caused by a bug. Please report this on http://github.com/colobot/colobot/issues
> including information on what you were doing before this happened and all the information below.
> ==============================
> You seem to be running a custom compilation of version 0.1.6-alpha, but please verify that.
> 
> The game was in phase PHASE_SIMUL (ID=7)
> Last started level was: category=missions chap=2 rank=1
> ==============================
```

Code:
```
public class Mover
{
	private int _subject = null;
	private int _object = null;
	
	void Mover()
	{}
	void SetSubject(int subject){
		this._subject = subject;
	}
	void SetObject(int obj){
		this._object = obj;
	}
	
	void loop(){
		while(true)
		{
			object subjectTarget = radar(this._subject);
			goto(subjectTarget.position);
			grab(InFront);
			object objectTarget = radar(this._subject);
			goto(objectTarget.position);
			drop(InFront);
		}
	}
}

extern void object::SwichCell()
{
	Mover m = new Mover();
	m.SetSubject(TitaniumOre);
	
}
```

**Comment by krzys-h** [comment]
The actual error is in these lines:
```C++
private int _subject = null;
private int _object = null;
```
You cannot assign null to an int variable. Use 0 instead, or just don't specify the default.

Classes in CBot are known to be terrible at error checking and usually crash instead. This is on our list of things to fix during CBot refactoring.

**Comment by krzys-h** [comment]
Fixed in #737
